README.md

```diff
@@ -78,3 +78,10 @@ Rspec/Focused:
 3. Commit your changes (`git commit -am 'Add some feature'`)
 4. Push to the branch (`git push origin my-new-feature`)
 5. Create a new Pull Request
+
+
+For running the spec files, this project depends on RuboCop's spec helpers. This means that in order to run the specs locally, you need a (shallow) clone of the RuboCop repository:
+
+```bash
+git submodule update --init --depth 1 vendor/rubocop
```

Comment:
@ybart Thanks for your contribution!

I am having difficulties with checking out the submodule by following this instruction. Here's what I did:

```bash
$ git clone https://github.com/lovewithfood/rubocop-rspec-focused.git
Cloning into 'rubocop-rspec-focused'...
remote: Counting objects: 95, done.
remote: Total 95 (delta 0), reused 0 (delta 0), pack-reused 95
Unpacking objects: 100% (95/95), done.
Checking connectivity... done.

$ cd rubocop-rspec-focused/

$ hub checkout https://github.com/lovewithfood/rubocop-rspec-focused/pull/3
Updating Agorize
remote: Counting objects: 18, done.
remote: Compressing objects: 100% (7/7), done.
remote: Total 18 (delta 2), reused 2 (delta 2), pack-reused 8
Unpacking objects: 100% (18/18), done.
From git://github.com/Agorize/rubocop-rspec-focused
 * [new branch]      autocorrect -> Agorize/autocorrect
Branch Agorize-autocorrect set up to track remote branch autocorrect from Agorize by rebasing.
Switched to a new branch 'Agorize-autocorrect'

$ git submodule update --init --depth 1 vendor/rubocop
Submodule 'vendor/rubocop' (git://github.com/bbatsov/rubocop.git) registered for path 'vendor/rubocop'
Cloning into 'vendor/rubocop'...
remote: Counting objects: 817, done.
remote: Compressing objects: 100% (732/732), done.
remote: Total 817 (delta 171), reused 189 (delta 79), pack-reused 0
Receiving objects: 100% (817/817), 1.34 MiB | 860.00 KiB/s, done.
Resolving deltas: 100% (171/171), done.
Checking connectivity... done.
fatal: reference is not a tree: 056439490c19ba5e00d16c7206483b73386c788f
Unable to checkout '056439490c19ba5e00d16c7206483b73386c788f' in submodule path 'vendor/rubocop'

$ git status
On branch Agorize-autocorrect
Your branch is up-to-date with 'Agorize/autocorrect'.
Changes not staged for commit:
  (use "git add <file>..." to update what will be committed)
  (use "git checkout -- <file>..." to discard changes in working directory)
  (commit or discard the untracked or modified content in submodules)

	modified:   vendor/rubocop (new commits, modified content)

no changes added to commit (use "git add" and/or "git commit -a")

$ ls -A vendor/rubocop/
.git

$ git diff
diff --git a/vendor/rubocop b/vendor/rubocop
index 0564394..da1f7ab 160000
--- a/vendor/rubocop
+++ b/vendor/rubocop
@@ -1 +1 @@
-Subproject commit 056439490c19ba5e00d16c7206483b73386c788f
+Subproject commit da1f7ab7de9c2a8c28fdda7cdf4f07b44e59fcfe-dirty
```

But if I do `git submodule update` without `--depth 1`, it works fine:

```bash
$ git clone https://github.com/lovewithfood/rubocop-rspec-focused.git
Cloning into 'rubocop-rspec-focused'...
remote: Counting objects: 95, done.
remote: Total 95 (delta 0), reused 0 (delta 0), pack-reused 95
Unpacking objects: 100% (95/95), done.
Checking connectivity... done.

$ cd rubocop-rspec-focused/

$ hub checkout https://github.com/lovewithfood/rubocop-rspec-focused/pull/3
Updating Agorize
remote: Counting objects: 18, done.
remote: Compressing objects: 100% (7/7), done.
remote: Total 18 (delta 2), reused 2 (delta 2), pack-reused 8
Unpacking objects: 100% (18/18), done.
From git://github.com/Agorize/rubocop-rspec-focused
 * [new branch]      autocorrect -> Agorize/autocorrect
Branch Agorize-autocorrect set up to track remote branch autocorrect from Agorize by rebasing.
Switched to a new branch 'Agorize-autocorrect'

$ git submodule update --init vendor/rubocop
Submodule 'vendor/rubocop' (git://github.com/bbatsov/rubocop.git) registered for path 'vendor/rubocop'
Cloning into 'vendor/rubocop'...
remote: Counting objects: 41781, done.
remote: Compressing objects: 100% (63/63), done.
remote: Total 41781 (delta 29), reused 0 (delta 0), pack-reused 41718
Receiving objects: 100% (41781/41781), 10.79 MiB | 1.96 MiB/s, done.
Resolving deltas: 100% (29458/29458), done.
Checking connectivity... done.
Submodule path 'vendor/rubocop': checked out '056439490c19ba5e00d16c7206483b73386c788f'
```

Do I anything wrong or maybe the instruction should just be `git submodule update --init vendor/rubocop`?